### ISSUE TITLE
Can't build on OS X 11.5.2 or 12.0.1.  Same problem

### ISSUE THREAD

**Issue by PaulHaeberli** [patriciogonzalezvivo/glslViewer#251]
BTW: I love glslViewer!

I looked at

/Users/xxxx/Desktop/glslViewer-main/build/CMakeFiles/CMakeError.log

and it says this:

I do have current version of Xcode installed and running fine on both machines.

BTW: I love glslViewer! Thank you for making it available Patricio.

Paul

Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
Build flags:
Id flags:

The output was:
1
ld: library not found for -lc++
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Compiling the C compiler identification source file "CMakeCCompilerId.c" failed.
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
Build flags:
Id flags:

The output was:
1
ld: library not found for -lSystem
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by PaulHaeberli** [comment]
I think I had some other version of your library! My Error.

**Comment by PaulHaeberli** [comment]
On further investigation I guess I'm still having the same problem.  Have you tried comiling on OSX recently?

**Comment by patriciogonzalezvivo** [comment]
Hi! Sorry my bad, introduce floating textures but seams MacOS don't like it. I revert the change on MacOS until I figurate out why. Update glslViewer and ada and try again.

Thanks for the patience and for the heads up

**Comment by PaulHaeberli** [comment]
Ok. I’m sorry for my confusion about it working, and closing the issue, etc. 

 I had gone into an older source tree and that compile did go ok. Too many folders on my desktop.

**Comment by PaulHaeberli** [comment]
I tried your latest changes on OSX.


I get this error when do make:

-- Configuring incomplete, errors occurred!
See also "/Users/xxxx/Desktop/glslViewer-main/build/CMakeFiles/CMakeOutput.log".
See also "/Users/xxxx/Desktop/glslViewer-main/build/CMakeFiles/CMakeError.log".



This is what the CMakeError.log says:

Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" failed.
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
Build flags:
Id flags:

The output was:
1
ld: library not found for -lc++
clang: error: linker command failed with exit code 1 (use -v to see invocation)


Compiling the C compiler identification source file "CMakeCCompilerId.c" failed.
Compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
Build flags:
Id flags:

The output was:
1
ld: library not found for -lSystem
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by patriciogonzalezvivo** [comment]
Mmm... are you following this directions https://github.com/patriciogonzalezvivo/glslViewer/wiki/Compile-on-MacOS ?

**Comment by PaulHaeberli** [comment]
No - I was using the instructions here:


https://github.com/patriciogonzalezvivo/glslViewer/wiki/Compiling#e-compiling-on-macos

They are the first macos compilation instructions I found. They don't work.



I tried your instructions that I had not seen before.


https://github.com/patriciogonzalezvivo/glslViewer/wiki/Compile-on-MacOS

And they worked fine!! I could build it on an intel mac.


Maybe update the instructions shown here:


https://github.com/patriciogonzalezvivo/glslViewer/wiki/Compiling#e-compiling-on-macos

to make it easier for other macos folks?
